plugins/connectors/pyproject.toml

```diff
@@ -7,7 +7,9 @@ authors = [{ name = "Kevin Su", email = "pingsutw@users.noreply.github.com" }]
 requires-python = ">=3.10"
 dependencies = [
     "flyte",
-    "flyteidl"
+    "flyteidl",
```

Comment:
should we remove flyteidl?  it's already part of `flyte`.  And if we remove it from flyte one day then connectors shouldn't need it either right?